cmd/initialize.go

```diff
@@ -25,9 +29,34 @@ var initializeCmd = &cobra.Command{
 
 func initializeDatabase() {
 	runDatabaseEffects(func(db *gorm.DB) {
-		if db.Migrator().CreateTable(database.Tables...) != nil {
+		if db.AutoMigrate(&table.Workflow{}, &table.ScheduledWorkflow{}, &table.WorkflowSchedulerLock{}) != nil {
 			panic("Failed creating tables")
 		}
+
+		if db.AutoMigrate(&table.SchemaEvolution{}) != nil {
+			panic("Failed creating SchemaEvolution")
+		}
+		v1 := db.Where("schema_version = ?", "1").First(&table.SchemaEvolution{})
+		if v1.Error != nil && !errors.Is(v1.Error, gorm.ErrRecordNotFound) {
+			panic("Failed checking SchemaVersion 1")
+		}
+		if v1.RowsAffected != 1 {
+			// record created table.SchemaEvolution
+			db.Create(&table.SchemaEvolution{SchemaVersion: "1", ExecutedAt: time.Now()})
+		}
+
+		v2 := db.Where("schema_version = ?", "2").First(&table.SchemaEvolution{})
+		if v2.Error != nil && !errors.Is(v2.Error, gorm.ErrRecordNotFound) {
+			panic("Failed checking SchemaVersion 2")
+		}
+		if v2.RowsAffected != 1 {
+			db.Migrator().CreateIndex(&table.Workflow{}, "Name")
```

Comment:
what if we just use `db.AutoMigrate(&table.Workflow{})`?

Reply:
it works, it created the expected index for workflows combine columns on name artifact.